### ISSUE TITLE
Enhance 'run' details

### ISSUE THREAD

**Issue by zkendall** [redgoose/daikin-one#7]
It looks like this currently just displays run time per hour. 

My heat-pump also has a supplemental gas furnace, so I'd like to be able to see data for heat pump vs furnace separately.

**Comment by redgoose** [comment]
I also have an auxiliary furnace, but it hardly gets cold enough for it to switch over. Because of that, I have not had the chance to inspect the device info to see if there’s a flag or indicator for running off auxiliary heat.

**Comment by zkendall** [comment]
I went and gave this project a try: https://github.com/apetrycki/daikinskyport

I had to setup a HomeAssistant VM etc etc, but it works well and has a ton of properties configured. But I don't really want to run HA, so I'm still interested in enhancing this project instead.

A couple things that maybe useful that I learned from that project:
- It has working charts for furnace 'demand' and 'actual', so should be able to track the properties down.
- It uses % instead of binary time. This is much more informative since the devices are often more than binary. It appears my furnace is 2 stage (I see 50% or 100% usage) whereas my heat pump is variable (see attached, many levels of demand).
  - In contrast this repos binary reporting per time only shows my heatpump nearly constantly running. But there is a big difference between constantly running at 25% and 100%, so it would be nice to change the data to illuminate this detail.
- It appears to have higher resolution of data point. I'm not exactly sure what, I think I see data points as close to 3min together, but hard to tell from chart.


<img width="1495" alt="image" src="https://github.com/redgoose/daikin-one/assets/2762875/16d027bc-1f8a-4e57-abbd-b20ed8730532">

**Comment by zkendall** [comment]
Here's the values the other project is using for HA: https://github.com/apetrycki/daikinskyport/blob/master/custom_components/daikinskyport/daikinskyport.py#L222-L236

I think the core ones I'm interested in are
- Heat pump demand - ctOutdoorHeatRequestedDemand
- Cooling demand - ctOutdoorCoolRequestedDemand
- Indoor fan - ctIFCCurrentFanActualStatus
- Indoor furnace - ctIFCCurrentHeatActualStatus

Sooo what can we do about this?

A couple things that come to mind.
1. Start collecting the data. Storage is cheap; 1 year is only ~100k 5min datapoints. We could start by just being more liberal about what data is collected and saved to db. This could be just the ones I mention above, or maybe all of the ones in the other project? 
2. Provide ways to opt in/out of data fields. This could be done at logging time. We could add a config property that indicates which properties to save.
3. Find ways to manage data on charts. They are already kinda noisy and will be worse with more data. Probably easiest method is similar to #-2 provide ways to opt in/out of what fields are charted.

**Comment by redgoose** [comment]
yep those are the values I was looking at as well. I'm good with keeping it simple and just go with always saving them to the db.

**Comment by zkendall** [comment]
Great! What're your thoughts on migration/back-wards compatibility? 

I created a draft PR to start saving the new data here: https://github.com/redgoose/daikin-one/pull/11

Should we just update the base 0001.sql script with the new columns and expect a fresh DB for any users. Or do you want to maintain a separate 002.sql script like I have now and assume any user will have run that modification? Or do you want to have the CLI itself inspect the schema and apply any missing migrations? Or ?

An impact of this decision for example is do we need to support nulls? Currently PeriodData doesn't support null values for db columns.

**Comment by redgoose** [comment]
For simplicity, just update the 0001 script, not too many users yet I would guess.

If you want to use a 0002 script, populate the columns with a default before setting to not null in the script itself.

**Comment by zkendall** [comment]
Sweet that makes my life easier. I ended up pushing them into the 0001 script.

I think that's all that was remaining for that PR, so I took it out of draft. Lemme know if any other changes are needed on that one.
